serverless/.vscode/settings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "editor.tabSize": 2,
+  "editor.detectIndentation": false,
+  "search.exclude": {
+    "package-lock.json": true
+  },
+  "editor.defaultFormatter": "dbaeumer.vscode-eslint",
+  "editor.formatOnSave": false,
+  "editor.codeActionsOnSave": [
+    "source.addMissingImports",
+    "source.fixAll.eslint"
+  ],
+  "jest.autoRun": {
+    "watch": false // Start the jest with the watch flag
```

Comment:
json comments also here

Reply:
Fixed in [62680f9](https://github.com/fleekxyz/non-fungible-apps/pull/140/commits/62680f92bc6a10e769ee4314edfaafa191ee4194).